packages/common/container-definitions/src/audience.ts

```diff
@@ -28,4 +28,20 @@ export interface IAudience extends EventEmitter {
      * or undefined if the specified client isn't connected
      */
     getMember(clientId: string): IClient | undefined;
+
```

Comment:
I believe we have to separate this interface into two - one that is available to most parties (including runtime itself), and one - that is available to Container to manipulate it. Container runtime should not be in position to call such APIs as clear()